packages/api/tests/models/test-granules-model.js

```diff
@@ -449,46 +447,39 @@ test('scan() will translate old-style granule files into the new schema', async
   );
 });
 
-test('getGranulesForCollection returns the queue of granules', async (t) => {
-  // create granules with different collectionIds and statuses
-  const collectionIds = ['testGetGranulesForCollection___v1', 'testGetGranulesForCollection___v2'];
-  const granulesV1 = range(5).map(() =>
-    fakeGranuleFactoryV2({ collectionId: collectionIds[0], status: 'completed' }));
-  const granulesV1Failed = range(5).map(() =>
-    fakeGranuleFactoryV2({ collectionId: collectionIds[0], status: 'failed' }));
-  const granulesV2 = range(5).map(() =>
-    fakeGranuleFactoryV2({ collectionId: collectionIds[1], status: 'completed' }));
-
-  const granule = new Granule();
-  await granule.create(granulesV1.concat(granulesV1Failed).concat(granulesV2));
-
-  // verify the granules retrieved meet the criteria and are in the correct order
-
-  // get all completed v1 granules
-  const queueV1Completed = granule.getGranulesForCollection(collectionIds[0], 'completed');
-  const sortedV1CompletedGranIds = granulesV1.map((gran) => gran.granuleId).sort();
-  for (let i = 0; i < 5; i += 1) {
-    const gran = await queueV1Completed.peek(); // eslint-disable-line no-await-in-loop
-    t.is(gran.granuleId, sortedV1CompletedGranIds[i]);
-    t.is(gran.collectionId, collectionIds[0]);
-    queueV1Completed.shift();
-  }
+test('getGranulesForCollection() only returns granules belonging to the specified collection', async (t) => {
```

Comment:
This seems to be testing sorting as well. Is this tested somewhere else?

Reply:
Added test for that here: fef57a490